### PR TITLE
Enable preview environments

### PR DESCRIPTION
**Pull request by bttf**
See links below!

### CODE REVIEW

**Comment by bttf** [comment]
@growthbook/devs Tested and ready to go

File: .github/workflows/preview-closed.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    - name: Context
+      uses: okteto/context@latest
+      with:
+        url: ${{ secrets.OKTETO_URL }}
+        token: ${{ secrets.OKTETO_TOKEN }}
+
+    - name: Destroy preview environment
+      uses: okteto/destroy-preview@latest
+      with:
+        name: pr-${{ github.event.number }}-bttf
```
Comment (tinahollygb) [inline .github/workflows/preview-closed.yml:20]: Did we want a different suffix or do we need to use your username because you configured everything?

**Review by tinahollygb — APPROVED** [review]
Looks good, just one minor comment about the URL.

File: .github/workflows/preview-closed.yml (hunk 1 above)
Comment (bttf) [reply .github/workflows/preview-closed.yml:20]: Yeah - we tried differently but Okteto won't accept a suffix other than the github username that owns the account

**Pull request merged**